rts/at/tx/pom.xml

```diff
@@ -157,6 +166,15 @@
       <artifactId>jakarta.transaction-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <!-- Quarkus -->
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
```

Comment:
I think quarkus-undertow is already included in quarkus-resteasy, so the quarkus-undertow dependency in redundant.